joomla/joomla-cms
# Updating FOF to 2.3.5

Updates FOF (Joomla! RAD layer) to version 2.3.5
# Backwards compatibility

This version is fully backwards compatible
# Translation impact

None.
# Test instructions

Install the patch. Use Joomla!'s two factor authentication feature in User manager and the Post-installation Messages component. Everything should load just fine.
